crowdsec-docs/sidebars.js

```diff
@@ -366,7 +366,7 @@ module.exports = {
     {
       type: "category",
       label: "Integrations",
-      items: ["cti_api/integration_chrome", "cti_api/integration_misp", "cti_api/integration_opencti", "cti_api/integration_thehive", "cti_api/integration_paloalto_xsoar", "cti_api/integration_maltego"],
+      items: ["cti_api/integration_chrome", "cti_api/integration_misp", "cti_api/integration_opencti", "cti_api/integration_thehive", "cti_api/integration_paloalto_xsoar", "cti_api/integration_maltego", "cti_api/integration_splunksoar"],
```

Comment:
can you rebase and resolve conflict with the previously merged PR? 

Also can you make it so that the naming is similar ? integration_splunksoar vs integration_splunk_siem